vendor/routerrpc/router.proto

```diff
@@ -0,0 +1,834 @@
+syntax = "proto3";
```

Comment:
Not a blocker for this PR, but how should we reason about versioning this file? All LND versions won't support all RPCs/messages that are defined here right? How do we figure out which version of LND we "support"?

Reply:
This is especially critical for our use case where we don't know the LND version being run by the node we need to connect from the gateway.

Proto3 has an [import capability](https://protobuf.dev/reference/protobuf/proto3-spec/#import_statement) which we could use to reference the protobuf definitions in LND, but we'd need an additional schema for determining what protobuf version to reference at compile time. This will dictate our minimum supported LND version

Investigating further